docs/disks/spec2006/runscript.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+cd /home/gem5/spec2006
+source shrc
+m5 readfile > workloads
+echo "Done reading workloads"
+if [ -s workloads ]; then
+    # if the file is not empty, run spec with the parameters
+    echo "Workload detected"
+    echo "Reset stats"
+    m5 exit
+
+    # run the commands
+    read -r workload size m5filespath < workloads
+    runspec --size $size --iterations 1 --config myconfig.cfg --noreportable --nobuild $workload
+    m5 exit
+    
+    # copy the SPEC result files to host
+    for filepath in /home/gem5/spec2006/result/*; do
+        filename=$(basename $filepath)
+        m5 writefile $filepath $m5filespath/$filename
+    done
+    m5 exit
+else
+    echo "Couldn't find any workload"
+    m5 exit
```

Comment:
Why are there three `m5 exit` calls?

Reply:
In the gem5 config script, I called m5.simulate() three times without checking whether the input is empty. I'll get rid of that, since in the launch script, we always have an input. So I'll will update the script.